assets/app/features/upgrade/templates/landing-page.jade

```diff
@@ -1,39 +1,69 @@
-            strong(translate='') upgrade.steps.landing.note
+                    span(translate='') upgrade.steps.landing.prechecks.form.check-again
 
-    button.btn.btn-success.center-block(
-        ng-disabled="!upgradeLandingVm.prechecks.completed || !upgradeLandingVm.prechecks.valid || upgradeLandingVm.prepare.running",
-        ng-class="{'spinner-visible': upgradeLandingVm.prepare.spinnerVisible, active: upgradeLandingVm.prepare.running}",
-        ng-click="upgradeLandingVm.beginUpgrade()"
-    )
-        suse-lazy-spinner(delay="2000", active="upgradeLandingVm.prepare.running",
-            visible="upgradeLandingVm.prepare.spinnerVisible")
-        span(translate='') upgrade.steps.landing.form.button
+            .upgrade.pull-left(ng-class="{'active': upgradeLandingVm.mode.valid, 'inactive': !upgradeLandingVm.mode.valid}")
+                crowbar-circle(ng-class="{'active': upgradeLandingVm.mode.valid}") 3
+                .h4.text-center(translate='') upgrade.steps.landing.upgrade.title
+                p(translate='') upgrade.steps.landing.upgrade.description
+                button.btn.btn-primary(
+                    ng-disabled="!upgradeLandingVm.prechecks.completed || !upgradeLandingVm.prechecks.valid || upgradeLandingVm.prepare.running || !upgradeLandingVm.mode.valid",
+                    ng-class="{'spinner-visible': upgradeLandingVm.prepare.spinnerVisible, active: upgradeLandingVm.prepare.running}",
+                    ng-click="upgradeLandingVm.beginUpgrade()"
+                )
+                    suse-lazy-spinner(delay="2000", active="upgradeLandingVm.prepare.running",
+                    visible="upgradeLandingVm.prepare.spinnerVisible")
+                    span(translate='') upgrade.steps.landing.form.button
+        .col-md-6
+            .mode.pull-left(ng-class="{'active': upgradeLandingVm.mode.active, 'inactive': !upgradeLandingVm.mode.valid}")
+                crowbar-circle(ng-class="{'active': upgradeLandingVm.mode.active}") 2
+                .h4.text-center(translate='') upgrade.steps.landing.mode.title
+                .initial
+                    p(
+                        translate='',
+                        ng-show='upgradeLandingVm.mode.type == nil'
+                    ) upgrade.steps.landing.mode.description
+                .normal
+                    strong
+                        p(
+                            translate='',
+                            ng-show='upgradeLandingVm.mode.type == "none"'
+                        ) upgrade.steps.landing.mode.normal.title
+                    button.btn.btn-primary(
+                        ng-disabled="upgradeLandingVm.mode.type != 'none' || upgradeLandingVm.mode.valid",
+                        ng-show="upgradeLandingVm.mode.type == 'none'"
+                        ng-click="upgradeLandingVm.continueNormal()"
+                    )
+                        span(translate='') upgrade.steps.landing.mode.normal.button
+                    p(
```

Comment:
I think more than 1 `p` is required, according to the comps. Please avoid using html tags in the language file